expenses-manager-frontend/src/main/frontend/src/app/services/category-service/category.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+import { Observable } from 'rxjs';
+
+export interface Category {
```

Comment:
Poti extrage interfata asta intr-un fisier common de interfete pentru ca il vom mai folosi si in alte parti.